docs/make.jl

```diff
@@ -5,7 +5,9 @@ makedocs(
     format=:html,
     pages=[
         "Home" => "index.md",
-        "API" => "pages/api.md"
+        "API" => "pages/api.md",
+        "Getting Started" => "pages/guide.md",
```

Comment:
I think you might as well put the contents of guide.md in index.md, as there is nothing there right now.